### ISSUE TITLE
Add an Option to Disable Loading Screen

### ISSUE THREAD

**Issue by TheFrozenDuck** [zellij-org/zellij#2024]
There should be an option to disable/hide the loading screen introduced by #1997 .

I have **alacritty** opening **zellij** on startup and it is kind of irritating when the loading screen message pops up every time a new terminal instance is opened.

**Comment by imsnif** [comment]
I'm open to this, but the loading screen is there to mask the 0.5 seconds it takes Zellij to query the terminal emulator for the default color palette and the pixel/cell ratio. Disabling it would also mean disabling this query, which would mean that some terminal apps might not function properly.

**Comment by TheFrozenDuck** [comment]
Disabling is the imporper term then and hiding would fit my idea much better.

My problem with the text is that it just flashes up really short and then is not seen anymore. The first few times it was kinda frustrating beacuse I did not grasp that it stems from **zellij** itself. Now it just annoys me a bit because it tends to grab a  second of attention, sometimes interrupting a bit.

I will look the next days into it and prepare a PR if I get to it.

**Comment by freerig** [comment]
Yep, an option like this would be nice. Can we also have an option to disable the stdin queries entirely, for those who delete the cache really often? Idk if stdin queries are essential, so then, if not, we should be able to enable/disable them.
Thx
